pkgs/development/python-modules/cachier/default.nix

```diff
@@ -0,0 +1,80 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, setuptools
+, watchdog
+, portalocker
+, pathtools
+, pytestCheckHook
+, pymongo
+, dnspython
+, pymongo-inmemory
+, pandas
+}:
+
+buildPythonPackage rec {
+  pname = "cachier";
+  version = "2.2.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-nm98LT87Z7yErKvIqMp93OEX9TDojqqtItgryHgSQJQ=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
```

Comment:
Why is setuptools propagated?

Reply:
They depend on it in [setup.py](https://github.com/python-cachier/cachier/blob/master/setup.py#L56) and I assumed it was because of `pkg_resources` or something to that end, but it seems they did it as a security measure. There is no mention of it in the build outputs, so I'm moving it to `pythonRemoveDeps`.